Content.Server/Administration/Systems/AdminVerbSystem.Antags.cs

```diff
@@ -101,5 +103,22 @@ private void AddAntagVerbs(GetVerbsEvent<Verb> args)
         };
         args.Verbs.Add(pirate);
 
+        Verb spaceNinja = new()
+        {
+            Text = "Make space ninja",
+            Category = VerbCategory.Antag,
+            // TODO: Change to ninja star?
```

Comment:
Localise.

Reply:
i was just copying the others i dont think they localised, should i do them too?

Reply:
also afaik they arent actually used ingame ever

Reply:
no response i guess resolved